sys/src/cmd/mv.c

```diff
@@ -53,6 +53,14 @@ main(int argc, char *argv[])
 		exits("bad usage");
 	}
 
+	/* Skip -f */
+	if(argv[1][0] == '-' && argv[1][1] == 'f' && argv[1][2] == 0) {
```

Comment:
It wasn't in original code because mv hasn't any flag on Plan 9. This ensures the first argument is nor passed as argument, so the classic and cool way... maybe is not appropiate. This is an ugly patch, officially mv hasn't flags. The way you like would be like:

    ARGBEGIN{
    case ’f’:
        break;
    default:
        usage();
    }ARGEND

Do you really want so? no problem for me, cp has it in that way because there were previous flags and I just added "p".